duckduckgo/macos-browser
# PacketTunnelProvider crash buttons

Task/Issue URL: https://app.asana.com/0/0/1205376937712482/f
Tech Design URL:
CC:

**Description**:

To verify that crash reporting works as expected from the PacketTunnelProvider extension, this PR adds debug menu items to trigger two types of crash: a fatal error and a memory overuse crash.

**Steps to test this PR**:
1. Go to Settings -> Debug -> Network Protection
2. Press one of the two crash simulation menu items (try both)
3. Go to Settings -> Network Protection and toggle the start switch
4. Confirm that it tried to connect then instantly fails (because 💥)

---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
